### DISCUSSION TITLE
Hot Reload VueJS

### DISCUSSION THREAD

**Discussion by buiphukhuyen** [OpnForm/OpnForm#178] [Q&A]
Hi,

I understand that Hot Reload is a powerful feature that can help streamline development. However, I'm not quite sure how to enable it within the OpnForm project.

Could you please provide guidance or steps on how to set up and use Hot Reload effectively in the OpnForm project? (I using Docker to deploy VPS and I want to translate language system to Vietnamese)

Thanks

**Comment by JhumanJ** [comment]
Hey, you should be able to do so by:
- Connect to the docker vm
- Run in the main folder (where the `package.json` file is), and run `npm run dev`

**Comment by buiphukhuyen** [comment]
<img width="1379" alt="Screenshot 2023-09-27 at 12 05 32" src="https://github.com/JhumanJ/OpnForm/assets/43597300/1f5f59fb-daef-401a-a363-c7cb0e6d9c71">
<img width="1377" alt="Screenshot 2023-09-27 at 12 05 23" src="https://github.com/JhumanJ/OpnForm/assets/43597300/f13869a0-c233-4c9a-a076-118d4d15dd79">

I followed your instructions but can't use it.
